.pre-commit-config.yaml

```diff
@@ -70,4 +70,5 @@ repos:
           -e, '*env*',
           -e, 'build/*'
         ]
+        exclude: '\w*pb2.py'
```

Comment:
Added this line since I had an issue with yapf failing with `yapf: input filenames did not match any python files` error. This error seems to occur when there are `.py` files to be committed but they are all also excluded from yapf by matching the `-e` flag patterns (so yapf has no valid python files to run on).